Gem/Swap/Scenes/SwapScene.swift

```diff
@@ -207,14 +240,26 @@ extension SwapScene {
 
     private func onChangeFromAsset(_: AssetData?, _: AssetData?) {
         model.resetValues()
+        model.resetSelectedProvider()
```

Comment:
```suggestion
        model.resetSelectedQuote()
```

Because there is possibility to have multiple quotes from the same provider